### DISCUSSION TITLE
Multi-line type definition in PHPDoc tags

### DISCUSSION THREAD

**Discussion by WinterSilence** [vimeo/psalm#8483] [Q&A]
Can I define type like as
~~~php
/**
* @psalm-return array{nodeType: string, text: string, line: int, filePos: int,
* tokenPos: int, endLine: int, endFilePos: int, endTokenPos: int}
*/
~~~
or definition must be single line?

**Comment by AndrolGenhald** [comment]
[Try it](https://psalm.dev/r/f957724674)! Yep, works fine! (so do trailing commas if you [format it nicely](https://psalm.dev/r/35ad56a16b))
